rktlet/runtime/rkt_runtime.go

```diff
@@ -74,7 +75,35 @@ func (r *RktRuntime) ContainerStatus(ctx context.Context, req *runtimeApi.Contai
 }
 
 func (r *RktRuntime) CreateContainer(ctx context.Context, req *runtimeApi.CreateContainerRequest) (*runtimeApi.CreateContainerResponse, error) {
-	return nil, fmt.Errorf("TODO")
+	// TODO(yifan): For now, let's just assume the podsandbox config is not used.
+	// TODO(yifan): More fields need to be supported by 'rkt app add'.
+
+	var imageID string
+
+	// Get the image hash.
+	imageName := *req.Config.Image.Image
+	resp, err := r.RunCommand("image", "fetch", "--store-only=true", "--full=true", "docker://"+imageName)
```

Comment:
@pskrzyns It's pulled from @tmrts‘ work. Not using image service because otherwise we will have another long running service.